server/jetty-app/build.gradle

```diff
@@ -57,6 +57,12 @@ if (!hasProperty('excludeS3')) {
     }
 }
 
+if (!hasProperty('excludeJson')) {
+    dependencies {
+        runtimeOnly project(':extensions-json-jackson')
```

Comment:
Should we pull in BSON support, too? Can't be expensive.

Reply:
I'm apt to not include it by default; I think we need to start the process of becoming more disciplined about optional dependencies.